tools/torch/main.lua

```diff
@@ -181,8 +181,8 @@ if opt.subtractMean ~= 'none' then
 end
 
 local classes
-local confusion
-local validation_confusion
+local trainConfusion
+local validConfusion
```

Comment:
question: don't you think the use of the contraction `valid` could be slightly ambiguous (given that `valid` is both an adjective and the contraction of `validation`?). `validationConfusion` sounds a bit wordy however I think this is unambiguous.